src/app/app.routing.ts

```diff
@@ -20,8 +19,9 @@ const APP_ROUTES: Routes = [
   { path: 'search-containers', loadChildren: 'app/containers/containers.module#ContainersModule' },
   { path: 'containers', loadChildren: 'app/containers/containers.module#ContainersModule' },
   { path: 'tools', loadChildren: 'app/containers/containers.module#ContainersModule' },
-  { path: 'search-workflows', loadChildren: 'app/workflows/workflows.module#WorkflowsModule' },
```

Comment:
Let's leave `search-workflows` here (in addition to `workflows`) in case anyone linked to our search for some reason